index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My porfolio</title>
+</head>
+<body>
+    <header>
+        <menu class="nav-container">
+            <h1><a href="#">Nana Antwi</a></h1>
```

Comment:
- [ ] @maccony202, it is not best practice to have two `<h1>` tags on the same HTML page. It is not good for SEO. Please refactor `<h1><a href="#">Nana Antwi</a></h1>` to `<a href="#">Nana Antwi</a>`.